packages/api/portfolioDrafts/application/createApplicationStep.test.ts

```diff
@@ -348,28 +348,29 @@ describe("Business rules validation tests", () => {
     expect(responseBody.details[0].message).toEqual("must NOT have less than 1 item");
     expect(result?.body).toMatch(/"\/body\/applications\/0\/environments"/);
   });
-  it("should return a validation error when admin roles are not acceptable", async () => {
-    const badAdminRolesRequest: ApiGatewayEventParsed<ApplicationStep> = {
-      body: mockApplicationsStepWithBadAdminRoles[0],
-      pathParameters: { portfolioDraftId: uuidv4() },
-    } as any;
-    const result = await handler(badAdminRolesRequest, {} as Context, () => null);
-    const responseBody = JSON.parse(result?.body ?? "");
-    expect(result?.statusCode).toEqual(ErrorStatusCode.BAD_REQUEST);
-    expect(responseBody.code).toMatch(/INVALID_INPUT/);
-    expect(responseBody.message).toMatch(/Invalid admin roles/);
-    expect(responseBody.error_map).toEqual({
-      acceptableAdministratorRoles: false,
-      applicationsWithNoAdmin: [0, 1],
-      environmentsWithNoAdmin: [
-        { appIndex: 0, envIndex: 0 },
-        { appIndex: 1, envIndex: 0 },
-      ],
-      hasAdminForEachApplication: false,
-      hasAdminForEachEnvironment: false,
-      hasPortfolioAdminRole: false,
-    });
-  });
+  // TODO(AT-6698): move to new operation that is implemented for Step 4 with adding operators
```

Comment:
The ticket I linked was not a new ticket. It was the old ticket responsible for adding team members. Please remove the TODO AT-6698.

Reply:
Replaced with `AT-?`. Thanks for clearing that up.

Reply:
New ticket is: AT-6835